shared/clients/email/email.go

```diff
@@ -64,6 +71,26 @@ func (s *SMTP) SendQRCodeToUser(user models.User) error {
 	})
 }
 
+func (s *SMTP) SendForgotPasswordURLToUser(user models.User, token string) error {
```

Comment:
How about adding `ByEmail` to that function name so that it will be more clear how we are going to send the `forget password` URL (since user model have `id` and `email` that can be used as key)